components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/listener/SCIMUserOperationListener.java

```diff
@@ -167,6 +169,10 @@ public boolean doPostSetUserClaimValueWithID(String s, UserStoreManager userStor
     public boolean doPreSetUserClaimValueWithID(String userID, String claimURI, String claimValue, String profileName,
                                                 UserStoreManager userStoreManager) throws UserStoreException {
 
+        // Validate whether claim update request is for a provisioned user.
+        if (isJitProvisionEnhancedFeature()) {
```

Comment:
`isJitProvisionEnhancedFeatureEnabled` is more meaningful IMO. WDYT?

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/367/commits/0652c627f1d971f39b773c2319554686a7f5218b